server/views/formBuilder.njk

```diff
@@ -231,6 +231,25 @@
                     }) }}
                 </div>
             {% endif %}
+            
+            {% if input.type === 'info'%}
+                <div id='info'>
+                    <h2 class='govuk-heading-s'>
+                        {{input.label}}
+                    </h2>
+                    <p>
+                      {{formValues}}
+                    </p>
+                    <input type="hidden" name="{{input.name}}" value="{{formValues}}" />
+                    <input hidden name="infoInputReviewed" value="true" />
+                    <h2 class="govuk-heading-m">Help with this licence condition</h2>
+                    {{ govukDetails({
+                        summaryText: input.helpLink.summary,
+                        text: input.helpLink.text,
+                        classes: 'govuk-!-margin-bottom-3'
+                    }) }}
+                </div>
+            {% endif %}
```

Comment:
should there be an end to end test to cover the behaviour of this kind of condition?

Reply:
By end to end I meant cypress test. Just as this has a lot of moving parts that affect the wider add conditions flow.

Does this change impact vary and licence version migration?

Reply:
There's now a cypress test to cover this!

Reply:
> Does this change impact vary and licence version migration?
It shouldn't, as we're just updating the existing licence policies to use the new condition config, rather than creating a new version of the policy. This shouldn't be a problem as we're removing an input, rather than adding a new one.